sympy/polys/tests/test_numberfields.py

```diff
@@ -637,9 +637,9 @@ def test_AlgebraicNumber():
 
     a = AlgebraicNumber(sqrt(2))
     b = to_number_field(sqrt(2))
-    assert a.args == b.args == (sqrt(2), Tuple())
+    assert a.args == b.args == (sqrt(2), Tuple(1, 0))
```

Comment:
Yes you'd have to change the `if coeffs !=` bit too.

How about `None` as the default input then? seems less confusing to me.  And then one could specify the zero element as `Tuple()` if desirable?